src/LogViewer/chunk-worker.js

```diff
@@ -127,6 +143,18 @@ const getParagraphClass = (lineNumber, { highlightStart, highlightEnd }) => {
     '';
 };
 
+// Tagged template function
+function escapeHtml(pieces) {
```

Comment:
@eliperelman This is a gem. I will follow these guidelines for future PRs as well. Thanks!